postgres/defaults.yaml

```diff
@@ -2,7 +2,7 @@
 
 postgres:
   use_upstream_repo: True
-  version: '9.5'
+  version: '10'
```

Comment:
Setting`10` as "default" should be okay. Debian stretch ships with version `9.6` but the `version` parameter is paired with `use_upstream_repo: True` and version `10` seems to be widely available upstream, and for [Debian stretch too](https://download.postgresql.org/pub/repos/apt/dists/stretch-pgdg/).  Approved.